src/components/matching/Matching.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { ReactElement } from 'react';
```

Comment:
Does this need to be in a `matching` directory since it's the only thing in there?